### DISCUSSION TITLE
Modification of pole figures

### DISCUSSION THREAD

**Discussion by mfuhr-mpg** [mtex-toolbox/mtex#1785] [Ask Anything]
Dear all,

I have a task where I could use some help from you guys.
I have orientation data in the form of EBSD orientation maps acquired from the cross-section of a strongly textured material. In a pole figure, this looks something like this:
![PF_MTEX](https://user-images.githubusercontent.com/79193413/229104893-67124274-1540-46bb-b86d-d2ccd0a3f363.jpg)
Now, I want to rotate the orientation data by 90° around the y-axis towards the x-axis (if you imagine a Cartesian sample coordinate system, where the x-axis points to the right, the y-axis to the top and the z-axis is pointing out of your screen). This simulates the measurement of the sample in a longitudinal section. The result should show two poles at phi = 0 | r=1 and phi = 180° | r=1 (when describing the pole figure as a 2D-object using polar coordinates).
In a second step, I would like to integrate the intensity at the equatorial plane (represented by a straight line in the following figure):
![PF_MTEX_line](https://user-images.githubusercontent.com/79193413/229122572-25808604-c5da-46ee-9299-b999702beb0f.jpg)

This is the (basic) code I used to draw the pole figure:
`
CS = crystalSymmetry('m-3m', [3.2 3.2 3.2], 'mineral', 'W', 'color', [0.53 0.81 0.98]);
%Load ebsd
[ebsd,~,diam,~] = EBSD_Master(Samples);
ori1=ebsd('W').orientations;
plotPDF(ori,Miller(1,1,0,CS),'contourf','antipodal','projection','eangle')
`

Thank you very much in advance for your help!
Best
Max

**Comment by lfgmorales** [comment]
Hi Max

I am not sure if that will do what you want, but if you use the command fiber as explained here

<https://mtex-toolbox.github.io/ODFPoleFigure.html>
Pole Figures | MTEX<https://mtex-toolbox.github.io/ODFPoleFigure.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/ODFPoleFigure.html>
        [favicon.png] <https://mtex-toolbox.github.io/ODFPoleFigure.html>


You may get your profile. For the rotation, simply (change angle and axis of rotation accordingly


ebsd = rotate(ebsd,rotation.byAxisAngle(xvector,180*degree))

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 31 Mar 2023, at 14:44, Maximilian Fuhr ***@***.***> wrote:

﻿

Dear all,

I have a task where I could use some help from you guys.
I have orientation data in the form of EBSD orientation maps acquired from the cross-section of a strongly textured material. In a pole figure, this looks something like this:
[PF_MTEX]<https://user-images.githubusercontent.com/79193413/229104893-67124274-1540-46bb-b86d-d2ccd0a3f363.jpg>
Now, I want to rotate the orientation data by 90° around the y-axis towards the x-axis (if you imagine a Cartesian sample coordinate system, where the x-axis points to the right, the y-axis to the top and the z-axis is pointing out of your screen). This simulates the measurement of the sample in a longitudinal section. The result should show two poles at phi = 0 | r=1 and phi = 180° | r=1 (when describing the pole figure as a 2D-object using polar coordinates).
In a second step, I would like to integrate the intensity at the equatorial plane (represented by a straight line in the following figure):
[PF_MTEX_line]<https://user-images.githubusercontent.com/79193413/229122572-25808604-c5da-46ee-9299-b999702beb0f.jpg>

This is the (basic) code I used to draw the pole figure:
CS = crystalSymmetry('m-3m', [3.2 3.2 3.2], 'mineral', 'W', 'color', [0.53 0.81 0.98]); %Load ebsd [ebsd,~,diam,~] = EBSD_Master(Samples); ori1=ebsd('W').orientations; plotPDF(ori,Miller(1,1,0,CS),'contourf','antipodal','projection','eangle')

Thank you very much in advance for your help!
Best
Max

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1785>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6EFZLV3REXKU24QTDW63GQTANCNFSM6AAAAAAWOTLFAM>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1785", "url": "https://github.com/mtex-toolbox/mtex/discussions/1785", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by mfuhr-mpg** [comment]
Hi Luiz!
Thank you very much, that helped me a lot.
Best
Max
